html5ever/fuzz/fuzz_targets/fuzz_document_parse.rs

```diff
@@ -29,7 +29,18 @@ fuzz_target!(|data: &[u8]| {
         return;
     };
 
-    let mut out = Vec::with_capacity(data.len());
+    struct Writer(usize);
+    impl std::io::Write for Writer {
+        fn write(&mut self, buf: &[u8]) -> Result<usize, std::io::Error> {
+            self.0 += buf.len();
+            Ok(buf.len())
+        }
+        fn flush(&mut self) -> Result<(), std::io::Error> {
+            Ok(())
+        }
+    }
+
+    let mut out = Writer(0);
```

Comment:
The total length counting appears to be unused. Could this be replaced with `std::io::sink`?

Reply:
Sure.